### PR TITLE
Thin Card

### PR DESCRIPTION
**Pull request by eddier**
Includes the thin card component along with the respective fragments.  Usage can be found in the storybook example and soon in the styleguide.

### CODE REVIEW

**Comment by eddier** [comment]
@jhegg I changed the name of the components to thin card from accordian for 2 reasons:
1.  we spelled it wrong...
2.  It didn't seem appropriate since some implementations of this component may have no accordion style functionality at all.

**Comment by jhegg** [comment]
Oh shoot, we'll need to convert the storybook stuff over to styleguidist, since that was also removed from master.

**Comment by eddier** [comment]
I merged master to this branch, you mentioned quite  a few items on the other components but those wont be addressed in this PR.  Those will be captured in the PR for those respective components.

**Comment by wa11-e** [comment]
Woohoo! new surge deployment available for viewing! :tada: [surgereview23518791.surge.sh](http://surgereview23518791.surge.sh)

**Comment by eddier** [comment]
@cdaringe this fixes the linting issues across the components.  Note that the parser was updated to support our static declarations.  https://github.com/feross/standard/issues/726

Can you review & merge so we can push an updated master to other feature branches?

File: src/components/LargeCard/LargeCardAction.jsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const LargeCardAction = (props) => {
+  return (
+    <a className={'fullview__goto__button button buttton-rect'} onClick={props.onClick}>
```
Comment (cdaringe) [inline src/components/LargeCard/LargeCardAction.jsx]: bump!

File: package.json [hunk 2]
```diff
@@ -78,7 +79,8 @@
       "lib",
       "semantic",
       "styleguide"
-    ]
+    ],
+    "parser": "babel-eslint"
```
Comment (cdaringe) [inline package.json]: cool. didnt know that this was a thing™

File: src/components/LargeCard/LargeCardRecentList.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <Flexbox className={`text-uppercase text-small state__body-${item.state}`} alignItems='center'marginRight='20px'>
+              {item.status}
+            </Flexbox>
+            <Flexbox className={`text-uppercase text-small state__body-${item.state}`} alignItems='center'>
+              {item.date}
+            </Flexbox>
+          </Flexbox>
+        )
+      })
+                }
```
Comment (cdaringe) [inline src/components/LargeCard/LargeCardRecentList.jsx]: if the linter takes it ill take it, but what's this little `}` guy doin' out here?

**Comment by cdaringe** [comment]
@antoinejuhel, check out the `ThinCard`.  see @wa11-e's post above.

@eddier nice!  this is cool stuff!  I had a few remarks above.  I have additional remarks here:

1. In your search what did you find about Styleguides docs on static members?  I ask because I'm not fully understanding this bit:
screen shot 2017-03-16 at 7 25 25 pm
1. `ThinCardWidget`.  I hate to complain about names (I really genuinely do) but I'm not understanding what the Widget is for/does?  He didn't have an example to look at.

What are your thoughts?  Thx!

**Comment by eddier** [comment]
I'm open to changing to the naming to something else.  The widget is a container for the key/value display and controls the styling of the left or right border.  There is an example on styleguidist included that shows you the implementation of the widget.

**Comment by cdaringe** [comment]
im cool w/ merging this soon, but the class typo should probably be 👀 'd at first.

spit balling on names (which have to happen here):

- `<ThinCard.Stat[istic] label={label} value={value} />` :)
- `<ThinCard.Column key={key} value={value} />` :/
- `<ThinCard.KeyValue ... />` :/

naming stuff stinks.

**Comment by eddier** [comment]
I'm ok with the first approach.  I'll update the component to reflect that syntax.

The large card component is a bit different on my local branch now, but I can send that up later today which obviously addresses the typo.

**Comment by cdaringe** [comment]
alrighty then.  in @eddier we trust! :)

**Pull request merged**